src/extensions/devel/ElectrodeGeometryTest/ElectrodeGeometry.tsx

```diff
@@ -167,4 +167,12 @@ const ElectrodeGeometry = (props: ElectrodeGeometryProps) => {
     )
 }
 
+function getMin(arr: number[]) {
+    return arr.reduce((max: number, v: number) => max <= v ? max : v, Infinity);
+}
+
+function getMax(arr: number[]) {
+    return arr.reduce((max: number, v: number) => max >= v ? max : v, -Infinity);
+}
+
```

Comment:
These two functions have shown up in several different files now, it'd be a good idea to put them in a library

Reply:
i agree. submitting an issue.